javaparser-core/src/main/java/com/github/javaparser/ast/type/UnionType.java

```diff
@@ -32,6 +30,7 @@ public UnionType(NodeList<ReferenceType<?>> elements) {
     }
 
     public UnionType setElements(NodeList<ReferenceType<?>> elements) {
+        notifyPropertyChange("elements", this.elements, elements);
```

Comment:
It is used once in JavaParser, but I expect that users will also use this text. I would personally feel a lot safer having this statically checked instead of equals'ed at runtime.